sys/linux/dev_camx.txt.const

```diff
@@ -1,45 +1,45 @@
 # Code generated by syz-sysgen. DO NOT EDIT.
-arches = arm64
+arches = arm, arm64
 AT_FDCWD = ???
-CAM_HANDLE_USER_POINTER = arm64:1
-CAM_MEM_FLAG_CACHE = arm64:1024
-CAM_MEM_FLAG_CDSP_OUTPUT = arm64:4096
-CAM_MEM_FLAG_CMD_BUF_TYPE = arm64:64
-CAM_MEM_FLAG_HW_READ_ONLY = arm64:2
-CAM_MEM_FLAG_HW_READ_WRITE = arm64:1
-CAM_MEM_FLAG_HW_WRITE_ONLY = arm64:4
-CAM_MEM_FLAG_KMD_ACCESS = arm64:8
-CAM_MEM_FLAG_PACKET_BUF_TYPE = arm64:512
-CAM_MEM_FLAG_PIXEL_BUF_TYPE = arm64:128
-CAM_MEM_FLAG_PROTECTED_MODE = arm64:32
-CAM_MEM_FLAG_STATS_BUF_TYPE = arm64:256
-CAM_MEM_FLAG_UMD_ACCESS = arm64:16
+CAM_HANDLE_USER_POINTER = 1
+CAM_MEM_FLAG_CACHE = 1024
+CAM_MEM_FLAG_CDSP_OUTPUT = 4096
+CAM_MEM_FLAG_CMD_BUF_TYPE = 64
+CAM_MEM_FLAG_HW_READ_ONLY = 2
+CAM_MEM_FLAG_HW_READ_WRITE = 1
+CAM_MEM_FLAG_HW_WRITE_ONLY = 4
+CAM_MEM_FLAG_KMD_ACCESS = 8
+CAM_MEM_FLAG_PACKET_BUF_TYPE = 512
+CAM_MEM_FLAG_PIXEL_BUF_TYPE = 128
+CAM_MEM_FLAG_PROTECTED_MODE = 32
+CAM_MEM_FLAG_STATS_BUF_TYPE = 256
+CAM_MEM_FLAG_UMD_ACCESS = 16
 CAM_MEM_HW_SHARED_ACCESS = ???
```

Comment:
Why is this not present? Misspelled? Should it be removed from txt file to not confuse reader?